lib/validate_encrypt_utils.pm

```diff
@@ -184,9 +184,6 @@ Where C<%args> expects the following parameters:
 =item C<$backup_file_info>
   unique string to match with the info of the backup file
 
-=item C<$backup_path>
```

Comment:
I don't understand why you want to remove it? Let's either keep it and have it in the test data, or let's remove it completely, so it's hard-coded.

Reply:
restored